deployment/helm/posix-mapper/values.yaml

```diff
@@ -1,24 +1,29 @@
 kubernetesClusterDomain: cluster.local
 
-# Tell Kubernetes to spin up multiple instances.  Defaults to 1.
+# -- Replica count for the service, default is 1
 replicaCount: 1
 
-# It's best to keep these set as such, unless you're willing to change these in several places.
+
 skaha:
+  # -- The namespace for the service, **Note:** This namespace is common between the Skaha and POSIX Mapper services. Do not change.
   namespace: skaha-system
 
 # POSIX Mapper web service deployment
 deployment:
-  hostname: example.host.com    # Change this!
+  # -- Hostname for the service, **Note:** Change this to your domain.
+  hostname: example.host.com
   posixMapper:
+    # -- The image for the service
     image: images.opencadc.org/platform/posix-mapper:0.2.1
+    # -- The image pull policy for the service
     imagePullPolicy: Always
+    # -- IVOA Resource ID for the service
     resourceID: ivo://opencadc.org/posix-mapper
 
-    # URI or URL of the OIDC (IAM) server.  Used to validate incoming tokens.
+    # -- URI or URL of the OIDC (IAM) server.  Used to validate incoming tokens.
     oidcURI: https://ska-iam.stfc.ac.uk/
```

Comment:
This should be relevant only to the deployer, so please replace ska-iam.stfc.ac.uk with iam.example.org.